README.md

```diff
@@ -1,70 +1,137 @@
-# Cluster Version Operator (CVO)
-
-## Building and Publishing CVO
-
-```sh
-./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.sh
+# Cluster Version Operator
+
+The Cluster Version Operator (CVO) is one of the [Cluster Operators](https://github.com/openshift/enhancements/blob/master/dev-guide/operators.md#what-is-an-openshift-clusteroperator)
+that run in every OpenShift cluster. CVO consumes an artifact called a _"release
+payload image,"_ which represents a given version of OpenShift and contains the
+manifests needed for all Cluster Operators to function. CVO reconciles
+the resources in the cluster to match the manifests in the release payload
+image. This way, CVO also implements _cluster upgrades_; after being provided
```

Comment:
The sentence "This way, CVO also implements..." is too long IMO. I suggest rephrasing a bit :) 

Maybe: 
The Cluster Version Operator (CVO) is one of the [Cluster Operators](https://github.com/openshift/enhancements/blob/master/dev-guide/operators.md#what-is-an-openshift-clusteroperator) that run in every OpenShift cluster. CVO consumes an artifact called a "release payload image," which represents a given version of OpenShift. It also contains the manifests needed for all Cluster Operators to function, reconciling the resources within the cluster to match these manifests in the resource payload image. 
As a result, CVO implements cluster upgrades. After being provided a release payload image for a newer OpenShift version, CVO reconciles all Cluster Operators to their updated versions and the Cluster Operators similarly update their operands.

Reply:
Thanks! I tried to apply your rephrasing in the current version while merging it with Trevor's suggestion above for the same paragraph.